src/app/branded/branded-checkout.tpl.html

```diff
@@ -16,7 +16,10 @@
           show-cover-fees="$ctrl.showCoverFees"
           next="$ctrl.next()"
           on-payment-failed="$ctrl.onPaymentFailed($event.donorDetails)"
-          radio-station-api-url="$ctrl.radioStationApiUrl">
+          radio-station-api-url="$ctrl.radioStationApiUrl"
+          use-v3="$ctrl.useV3"
```

Comment:
I just finished adding some comments about making sure to say `$ctrl.useV3 === 'true'` instead of just checking `$ctrl.useV3`. However, if we make this change here, then we know that `useV3` is a bool instead of a string in `branded-checkout-step-1`, which will simplify all of those checks. Then it will be safe to just check `$ctrl.useV3` or `!$ctrl.useV3`.

```suggestion
          use-v3="$ctrl.useV3 === 'true'"
```